_posts/this-week-in-solid/2019-01-01-this-week-in-solid.md

```diff
@@ -13,7 +14,7 @@ You can always check here for the most current issue as well as find a record of
 
 # Next Up
 
-We're currently working on next week's edition. You can contribute by [making a Pull Request](https://github.com/solid/solidproject.org/edit/master/_posts/this-week-in-solid/next.md)!
+We're currently working on next month's edition. You can contribute by [making a Pull Request](https://github.com/solid/solidproject.org/edit/master/_posts/this-week-in-solid/next.md)!
```

Comment:
```suggestion
We're currently working on next month's edition. You can contribute by [making a Pull Request](https://github.com/solid/solidproject.org/edit/master/_posts/this-month-in-solid/next.md)!
```
I see that you changed `_posts/this-week-in-solid/next.md` to replace `TWiS` by `TMiS`, do you think we should create a new `_posts/this-month-in-solid/` directory ? `_posts/this-week-in-solid/` would archive previous content, and `_posts/this-month-in-solid/` would contain the new issues.

Another option is to change the existing directory name to `_posts/newsletter/`, which would be agnostic to the periodicity.

Reply:
That's a good question. I left the non-visible references to weekly updates in place to minimise the number of places where it could break, but that can certainly get confusing in situations such as this. I think renaming it to newsletter is a good idea, but preferably after merging this, so that we can first verify on solid.github.io whether this change doesn't introduce any problems. What do you think?

(Goes to show that, no matter how many abstractions you think of, there's always going to be a change needed in the future that you did not anticipate. Or at least, I didn't.)

Reply:
Merging this, checking that redirections work properly, and only then renaming the directory seems like a reasonable and safe approach. And 100%, there's always a breaking change waiting for you :)